io.gitlab.construo.construo.json

```diff
@@ -0,0 +1,32 @@
+{
+    "app-id": "io.gitlab.construo.construo",
+    "runtime": "org.freedesktop.Platform",
+    "runtime-version": "1.6",
+    "sdk": "org.freedesktop.Sdk",
+    "rename-icon": "construo",
+    "rename-desktop-file": "construo.x11.desktop",
+    "rename-appdata-file": "construo.appdata.xml",
+    "finish-args": [
+        "--device=all",
+        "--share=ipc",
+        "--socket=x11",
+        "--socket=wayland"
```

Comment:
Does this actually work under wayland only? If so, cool!

Reply:
Won't work right now, but there is an OpenGL renderer in there, so should be doable. Just have to do some more upstream work on it.

Reply:
Sure - remove that permission then and it'll use the XWayland fallback rather than just crashing.